packages/nx/src/config/relative-cwd.ts

```diff
@@ -0,0 +1,5 @@
+import { relative } from 'path';
+
+export function relativeCwd(cwd: string, root: string): string | null {
+  return relative(root, cwd).replace(/\\/g, '/') || null;
```

Comment:
`relativeCwd` is basically the same as `relative` now.

Do you know when it actually matters that this is normalized? For the usages above, I don't think it is important. For this function, I feel like we can just inline this where it is used rather than keeping it a separate function.

Reply:
i trace the code, relative path will be passed to convertSmartDefaultsIntoNamedParams, which does normalizing there. so we don't need to normalize it here.